src/server/scripts/Northrend/VaultOfArchavon/boss_emalon.cpp

```diff
@@ -246,7 +246,7 @@ class npc_tempest_minion : public CreatureScript
                         if (OverchargedTimer <= diff)
                         {
                             DoCast(me, SPELL_OVERCHARGED);
-                            OverchargedTimer = 2000;
+                            OverchargedTimer = 2s;
```

Comment:
this is wrong because overcharged is declared as uint32

Reply:
reverting this particular line edit in https://github.com/TrinityCore/TrinityCore/pull/22415/commits/5fee827d8444a5f0d1f34a31025e8eca5c05eb4a